tanzu-build-service/install-tbs.hbs.md

```diff
@@ -154,74 +167,6 @@ to your `tbs-values.yaml` file. This is to exclude the default `lite` dependenci
 1. If you configured `full` dependencies in your `tbs-values.yaml` file, install the `full` dependencies
 by following the procedure in [Install full dependencies](#tap-install-full-deps).
 
-## <a id='alternative-creds'></a> (Optional) Alternatives to plaintext registry credentials
-
-Tanzu Build Service requires credentials for the `kp_default_repository` and the Tanzu Network registry.
-
-You can apply them directly in-line in plaintext in the `tbs-values.yaml` or `tap-values.yaml`
-configuration by using the `kp_default_repository_username`, `kp_default_repository_password`, `tanzunet_username`,
-and `tanzunet_password` fields.
-
-If you do not want credentials saved in plaintext, you can use existing secrets or IAM roles by using
-[secret references](#install-secret-refs) or [AWS IAM authentication](#tbs-tcli-install-ecr)
-in your `tbs-values.yaml` or `tap-values.yaml`.
-
-### <a id='install-secret-refs'></a> Use Secret references for registry credentials
```

Comment:
Link will break on [line 70](https://github.com/pivotal/docs-tap/blob/1ac74b9e0c554542a6a7eafd51026c67f4810f96/multicluster/reference/tap-values-build-sample.hbs.md?plain=1#L70) of `multicluster/reference/tap-values-build-sample.hbs.md`.

Reply:
Here to: https://github.com/pivotal/docs-tap/blob/1ac74b9e0c554542a6a7eafd51026c67f4810f96/multicluster/reference/tap-values-iterate-sample.hbs.md?plain=1#L70